app/models/calculationList/CalculationListModel.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.calculationList
+
+import play.api.libs.json.{Format, Json}
+
+sealed trait CalculationListResponseModel
+
+case class CalculationListModel(calculationId: String,
+                                calculationTimestamp: String,
+                                calculationType: String,
+                                crystallised: Option[Boolean]) extends CalculationListResponseModel
```

Comment:
I presume None here maps to false? If so, can we flatten this to just Boolean?

Reply:
It throws an ISE at the moment
This is an optional field in both calc APIs